src/mlpack/tests/ann_regularizer_test.cpp

```diff
@@ -0,0 +1,118 @@
+/**
+ * @file ann_regularizer_test.cpp
+ * @author Saksham Bansal
+ *
+ * Tests the ann regularizer modules.
```

Comment:
This should be `ANN` in place of `ann`.

Reply:
@saksham189 Can you make this change? I'll merge this in then?

Reply:
sorry for the delay.